src/powershell-native/nativemsh/pwrshcommon/ClrHostWrapper.h

```diff
@@ -1,7 +1,7 @@
 // ----------------------------------------------------------------------
 //
 //  Microsoft Windows NT
-//  Copyright (C) Microsoft Corporation, 2014.
+//  Copyright (c) Microsoft Corporation, 2014.
```

Comment:
I believe we use common pattern "Copyright (c) Microsoft.  All rights reserved."